include/fty_common_utf8.h

```diff
@@ -31,17 +31,27 @@ extern "C" {
 //  Self test of this class
 void fty_common_utf8_test (bool verbose);
 
+// C wrapper for UTF8::escape
+char* escape(const char *string);
+
+#ifdef __cplusplus
+}
+#endif
+
 namespace UTF8 {
 
 int8_t utf8_octets (const char *c);
 
 int utf8_to_codepoint(const char *uchar, char **codepoint);
 
 int utf8eq(const char *s1, const char *s2);
-}
 
-#ifdef __cplusplus
+// Convenient wrapper for escape"("const char *string")"
+std::string escape(const char *string);
```

Comment:
Just to check, is this file included in any C source file? If so, I think that std::string will be visible to C, but not defined.